lib/Sema/CSDiagnostics.cpp

```diff
@@ -908,6 +908,30 @@ bool MissingAddressOfFailure::diagnoseAsError() {
   return true;
 }
 
+SourceLoc InvalidUseOfAddressOf::getLoc() const {
```

Comment:
Same thing as in `CSFix.cpp` I think it should be left in place, it might make it more difficult to blame the changes later on.